test/test_profiler.py

```diff
@@ -99,6 +101,70 @@ def forward(self, x):
 
         torch._C._set_graph_executor_optimize(prev_opt)
 
+    def test_python(self):
```

Comment:
need another test case to ensure optimizer is not double hooked?

Reply:
Thanks. Just tested offline.